### DISCUSSION TITLE
Boofuzz multiple targets

### DISCUSSION THREAD

**Discussion by ttufts** [jtpereyda/boofuzz#677] [General]
The documentation for the `add_target` method of `Session` indicates that multiple targets will be parallelized when fuzzing. In practice it seems that only the first target is ever used. Some quick review of the code shows that `target[0]` is used all through the `Session` class. Is this an unfinished feature or am I just missing something?

**Comment by SR4ven** [comment]
You are right, multiple targets were planned but are not supported at this point.

**Comment by ttufts** [comment]
I forked and made some changes that seem to have solved it in my fork. When
I’m done testing I’ll send a pull request.

**Reply by SR4ven** [reply]
Cool! Looking foreward to the PR.
